compiler/rustc_codegen_llvm/src/gotoc/intrinsic.rs

```diff
@@ -217,6 +217,11 @@ impl<'tcx> GotocCtx<'tcx> {
             }};
         }
 
+        if let Some(stripped) = intrinsic.strip_prefix("simd_shuffle") {
```

Comment:
Could this be integrated into the match below instead of its own code block?

```rust
i if i.strip_prefix("simd_shuffle") => ...
```
```

Reply:
Apparently this is  an "if let guard" and is marked as experimental currently: https://github.com/rust-lang/rfcs/blob/master/text/2294-if-let-guard.md

I'm going to leave it as-is for now.

Reply:
Why not have it at the end of the match with an `starts_with` guard?

Reply:
Still, you can match the prefix, then extract the suffix before calling the codegen function.